src/plugin/helpers/plugin.ts

```diff
@@ -23,7 +23,14 @@ export const getElementFieldViewFromType = (
     case "text":
       return new TextFieldView(node, view, getPos, offset, innerDecos);
     case "richText":
-      return new RichTextFieldView(node, view, getPos, offset, innerDecos);
+      return new RichTextFieldView(
+        node,
+        view,
+        getPos,
+        offset,
+        innerDecos,
+        field.createPlugins ? field.createPlugins(node.type.schema) : []
```

Comment:
We could move the example setup plugins here as a default?

Reply:
Coooould do! I'd be tempted to move `exampleSetup` into devDependencies and avoid bundling it in the library ... but it's probably quite little.

Reply:
On reflection, I think if we decide we'd like to provide it, let's make it a field creator re: #54 (for example, `createDefaultRichTextField` or similar), to make sure plugin consumers that don't require it can tree shake that dependency out.

I do think that it makes sense to provide it down the line, b/c anyone playing around with this library will want a quick way to express an RTE, for sure.

Reply:
That's a great idea, we could rebase the field functions once they're merged and add the default one.